packages/tezos-micheline/tezos-micheline.7.0/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+maintainer: "contact@tezos.com"
+authors: [ "Tezos devteam" ]
+homepage: "https://www.tezos.com/"
+bug-reports: "https://gitlab.com/tezos/tezos/issues"
+dev-repo: "git+https://gitlab.com/tezos/tezos.git"
+license: "MIT"
+depends: [
+  "tezos-tooling" { with-test & = version }
+  "dune" { >= "1.11" }
+  "tezos-error-monad" { = version }
+  "uutf"
+  "alcotest-lwt" { with-test & = "0.8.5" }
```

Comment:
This exact dependency will get problematic in the future. Do you have an upstream Tezos issue to support alcotest-lwt >= 1.0? There was an API change, but it's fairly straightforward to port older tests to the new API. cc @samoht @craigfe -- we'd be happy to help move that forward if you have any doubts.

Reply:
Tarides already proposed their helps, this will happen :-)

Reply:
(Alcotest port MR now exists here: https://gitlab.com/tezos/tezos/-/merge_requests/1814.)